docs/en/writing-running-appium/caps/index.html

```diff
@@ -2023,8 +2023,8 @@ <h3 id="general-capabilities">General Capabilities</h3>
 </tr>
 <tr>
 <td><code>udid</code></td>
-<td>Unique device identifier of the connected physical device</td>
-<td>e.g. <code>1ae203187fc012g</code></td>
+<td>Unique device identifier of the connected physical device. For Android, this is the device name as listed by <code>adb devices</code>.</td>
```

Comment:
I added this sentence because UDID is a term used for iOS devices: Android devices don't have a UDID.